### PR TITLE
[T4A7][W09-B2]Duan Yichen

### PR DESCRIPTION
**Pull request by Yichen-D**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/NameTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.data.person;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
```
Comment (damithc) [inline src/seedu/addressbook/data/person/NameTest.java:9]: The convention is to name the test class based on the class being tested. e.g. `PersonTest`

**Comment by damithc** [comment]
Good try, but you are a bit off from TDD. For example, a TDD session starts by adding a failing test, not by adding an empty method. Also, you TDD adds bit of test code and bit of functional code in tandem (you were supposed to commit after each step), not all add tests first. Please take a look at the course handouts and suggested readings given in the assignment description to get a good understanding of TDD.
Please close the PR after reading comments.
